ingestors/npm.go

```diff
@@ -38,6 +40,23 @@ func (ingestor *NPM) Name() string {
 	return "npm"
 }
 
+/**
+ * NPM package updates are ingested from a continuous reading of a remote
+ * CouchDB database at replicate.npmjs.com. CouchDB databases provide a Changes
+ * feed that provide the changes since the last set of changes were published:
+ * https://docs.couchbase.com/sync-gateway/current/changes-feed.html
+ *
+ * We connect to the CouchDB database and continually read for the next set
+ * of changes to the database. Once we receive changes, we take the found releases
+ * and add them to the processing queue. If no changes are available, we
+ * reconnect to the database in a number of seconds and try again.
+ *
+ * Since this is based on detecting changes to the NPM database, it may be
+ * possible for a client to miss out on changes for some reason. Libraries only
```

Comment:
@johnbintz-tidelift — Curious about whether you identified any of the possible reasons to miss updates, and how frequently that may happen?
Seems like there are multiple reports of similar issues (e.g.  https://github.com/npm/feedback/discussions/937 ) with no clear explanation as to why this may have happened.